examples/panel/stock-market-chatbot/app.py

```diff
@@ -0,0 +1,231 @@
+import panel as pn
+import pandas as pd
+from chat import  analyze_image_with_text
+from dotenv import load_dotenv
+import os
+from imagekitio import ImageKit
+import nest_asyncio
+
+from stock import  store_data_in_duckdb, get_data_from_duckdb, get_stock_symbols
+from bokeh.themes import Theme
+from bokeh.io import curdoc
+
+import plotly.express as px
+pd.options.plotting.backend = "plotly"
+
+load_dotenv(".env")
```

Comment:
don't use .env files, show how to set them via `export NAME=value` when testing locally and explain how they can put the values when deploying the app

Reply:
resolved https://github.com/ploomber/doc/pull/124/commits/5134a06dd9966b18ae5cc687803e19a4ddf4f0c9